content/sensu-go/5.14/release-notes.md

```diff
@@ -63,18 +63,22 @@ See the [upgrade guide][1] to upgrade Sensu to version 5.14.0.
 - To facilitate discussion about feature requests, in the Web UI, feedback is directed to Discourse rather than the GitHub repository's issues page.
 - In the [web UI][80], when a user lands on a page inside a namespace that no longer exists or they do not have access to, the drawer opens to that namespace switcher to help clarify next steps.
 - Updated Go version from 1.12.3 to 1.13.1.
+- The GraphQL `putWrapped` mutation now accepts wrapped JSON with empty outer objectmeta.
 
 **FIXES:**
 
 - ([Licensed tier][79]) `sensuctl` can now create federation resources.
 - ([Licensed tier][79]) `sensuctl` on Windows can now create Postgres resources.
 - ([Licensed tier][79]) Fixed a bug that resulted in event metrics being ignored when using the Postgres store.
 - Fixed a bug that caused checks to stop executing after a network error.
+- Fixed a bug that prevented `sensuctl create` with `stdin` from working.
 - Splayed proxy checks are executed every interval (instead of every interval + interval * splay_coverage).
 - GraphQL ensures that proxy entity label and annotations are redacted.
```

Comment:
GraphQL isn't something users interact with directly, it exists primarily for the web UI to query the Sensu datastore. In order to be relevant to users, fixes to GraphQL generally need to be tracked back to a source issue. 

In this case I found that https://github.com/sensu/sensu-go/issues/3261 is closed by the change this line describes. I think this is a significant fix for many folks, as those who store a password or other secret in a label or annotation do not want see that secret exposed in the web UI.

```suggestion
- Proxy entity labels and annotations are now redacted in the web UI as expected.
```